src/core/Segments.tsx

```diff
@@ -37,12 +34,12 @@ const Segments = React.forwardRef<Line2, SegmentsProps>((props, forwardedRef) =>
   const [geometry] = React.useState(() => new LineSegmentsGeometry())
   const [resolution] = React.useState(() => new THREE.Vector2(512, 512))
 
-  const [positions] = React.useState(() => Array(limit * 6).fill(0))
-  const [colors] = React.useState(() => Array(limit * 6).fill(0))
+  const [positions] = React.useState<number[]>(() => Array(limit * 6).fill(0))
```

Comment:
was this not infered?

Reply:
No, because .fill doesn't change the return type of the array

Reply:
til